nservicebus/messaging/memory-transport.md

```diff
@@ -6,7 +6,10 @@ redirects:
  - nservicebus/using-the-in-memory-bus
 ---
 
-NOTE: Bus.InMemory feature has been deprecated starting from Version 4.6 and removed in Version 5.0
+{{NOTE: Bus.InMemory feature has been deprecated starting from Version 4.6 and removed in Version 5.0
+
+In Version 6, the `IBus` interface has been deprecated and removed. Use the `IMessageHandlerContext` interface instead. For more information, please consult the [upgrade guide](/nservicebus/upgrades/5to6.md).
```

Comment:
should that be two different notes?

also, sean's comment on mentioning bus session? (or avoiding IMessageHandlerContext)